### ISSUE TITLE
Decision on timelines & reminders for participants

### ISSUE THREAD

**Issue by fazil-ey** [alexandralloyd/DJ-program#28]
We want to ensure ADMs complete the process in time. To do so we have to decide on the timelines for filling the form, having the validation meeting and signing off. We also need to decide on the cadence for reminders. 

Decisions 
1. What is the deadline for 
    - ADMs to fill the form for validation meeting
    - signing off after the validation meeting

2. When should reminders be sent for each event
    - If automated, what should the timeline be, e.g. 5 days before deadline, 2 days before deadline
   - if manually done, who is responsible for sending the reminders

**Comment by HasanSifat** [comment]
- the decision on the form availability window. Decision on Tuesday 20 sept, 2022 @JodiLittle 

- Decision pending on sending auto-reminders (nice to have) Jodi- if possible then the answer is yes. Question- do we send reminders from teams or from the tool? @HasanSifat

**Comment by JodiLittle** [comment]
Timelines - SLR process will start November 15 and will remain open until  11:59 January 27, 2023. Ministries will manage their own validation approach and timing. They will manage their timing in accordance with the date of closure on Jan 27th. Decision about alerts to come.

**Comment by JodiLittle** [comment]
Reminders will be sent out January 16th (2 week reminder) and January 25 (2 day reminder) via email from the product owners.

**Comment by HasanSifat** [comment]
This is one ticket we can consider moving to DEV board to possibly automate the trigger to send out emails from the platform.
